.github/CONTRIBUTING.md

```diff
@@ -7,6 +7,14 @@
 Search [the issue tracker][issue tracker] to ensure that there is no open issue addressing the bug.
 If that is the case, [open a new issue][new issue]. Make sure to include a clear title, description, and as much relevant information as possible.
```

Comment:
The second sentence may not be so relevant anymore with the existence of templates. Try something like:
```
Use the following templates:
```